openassessment/xblock/data_conversion.py

```diff
@@ -57,6 +73,27 @@ def convert_training_examples_list_to_dict(examples_list):
     ]
 
 
+def update_assessments_format(assessments):
+    """
+    For each example update 'answer' to newer format.
+
+    Args:
+        assessments (list): list of assessments
+    Returns:
+        list of dict
+    """
+    for assessment in assessments:
+        if 'examples' in assessment:
+            for example in assessment['examples']:
+                if isinstance(example['answer'], unicode) or isinstance(example['answer'], str):
```

Comment:
Just curious why you're doing the isinstance check here. Would there be a scenario where it would not be unicode or str?